src/gen_tables/between.rs

```diff
@@ -58,15 +58,16 @@ pub fn gen_between() {
 }
 
 // Write the BETWEEN array to the specified file.
+#[allow(clippy::needless_range_loop)]
 pub fn write_between(f: &mut File) {
-    write!(f, "const BETWEEN: [[BitBoard; 64]; 64] = [[\n").unwrap();
+    writeln!(f, "const BETWEEN: [[BitBoard; 64]; 64] = [[").unwrap();
     for i in 0..64 {
         for j in 0..64 {
-            unsafe { write!(f, "    BitBoard({}),\n", BETWEEN[i][j].0).unwrap() };
+            unsafe { writeln!(f, "    BitBoard({}),", BETWEEN[i][j].0).unwrap() };
```

Comment:
As i understand, `unsafe` is not required here, right?

Reply:
It is required, since it's an access to a mutable static.

Reply:
Oh, I see. If I understand correctly there are two `BETWEEN`: const (in `magic_gen.rs`) and [static](https://github.com/jordanbray/chess/blob/0e538fbee63d01b282cd22a40f1b6655cd6922ab/src/gen_tables/between.rs#L9).

Then maybe move `unsafe` to smaller scope?
```rust
writeln!(f, "    BitBoard({}),", unsafe { BETWEEN[i][j] }.0).unwrap();
```